### DISCUSSION TITLE
geox vs rule-set: MetaCubeX对二者的态度？

### DISCUSSION THREAD

**Discussion by Paulgudring** [MetaCubeX/mihomo#949] [General]
隔壁世界似乎在singbox 1.8.0 pre-release 版本加入rule-set后逐渐去除geox的功能。我能够理解geox存在与原初目的不一致的问题，而rule-set更加灵活[^1]，但还是想求教一下是否有更重要的原因，毕竟可以保留功能供用户选择。

因为在mihomo的使用中，geox的规则使配置足够简洁、运行足够高效。尽管这部分的实现并不是以singbox为上游，我仍好奇metacubex团队的态度，我还是期待能够保留geox规则来处理大部分分流，而用rule-set来解决特殊的规则分流。

[^1]: 见https://sing-box.sagernet.org/deprecated/。

**Comment by Larvan2** [comment]
如非特殊情况（如外星人入侵），不会去除geox的支持
